doc/Makefile

```
@@ -26,6 +26,7 @@ help:
 clean:
 	rm -rf $(BUILDDIR)/*
 	rm -rf $(SOURCEDIR)/examples
+	rm -rf source/examples/gallery_examples
```

```suggestion
	rm -rf $(SOURCEDIR)/examples/gallery_examples
```

Also, why is this duplicated?

---

Fixed as per suggestions.